10-JavaScript/fun-with-functions.md

```diff
@@ -133,15 +133,15 @@ data.forEach(function (num) {
 });
 
 let average = sum / data.length;
-console.log("Average is " + average);
+console.log(`Average is ${average}`);
 ```
 
 ### Functional Expressions vs. Function Declarations
```

Comment:
In this section we are using the phrase "functional expression" however I believe the correct terminology is just "function expression".

Reply:
Shortly after this section, in the paragraph after the code snippet for the function declaration example, we say to always use `let` when defining functions and we should probably say `const` instead. (I can't comment on that line directly because it was not changed.)